0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -52,7 +59,72 @@ Content-Type: application/octet-stream
 < Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
 ```
 
-All ILP Packets MUST be returned with the HTTP status code `200: OK`.
+If the request included a `Request-Id` header, the recipient handling the ILP Prepare SHOULD choose to handle the packet asynchronously and return the corresponding ILP Fulfill/Reject in a separate outgoing HTTP request.
+
+In the asynchronous mode, if the request is semantically valid, the recipient MUST respond immediately that the ILP Prepare is accepted for processing, even if the packet will ultimately be rejected:
+
+```http
+HTTP/x.x 202 Accepted
+```
+
+#### Retry Behavior
+
+Two peers MAY retry ILP Prepare packets, but MUST both agree to support retries out-of-band to prevent duplicate side effects.
+
+If the recipient of the ILP Prepare responds with an `5xx` or `409 Conflict` status, or no HTTP response is received within a given timeout, the sender of the ILP Prepare SHOULD retry sending the packet with the same idempotency key.
+
+The sender MUST conclude retrying after receiving a response with a `2xx` status, `4xx` error, or receiving the corresponding ILP Fulfill/Reject packet.
```

Comment:
`4xx` error other than `409`. also what does the 409 conflict signify in this flow? It seems to have special meaning but is not fully explained

Reply:
Eh, maybe I should just remove retrying on `409`? (Stripe uses `409` for idempotent retries, but I'm not sure it matters)

Reply:
Removed retrying on `409`: https://github.com/interledger/rfcs/compare/09916ea8c1232f846610940066298f052f140c8f...8b43dafce2c84d073bc70afacf396f700d4ff8f5#diff-ccfda9b7df50a9cfd8618bb9c08e360aR74